<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>362</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Rounded MT Bold</family>
    <pointsize>13</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>NoGo Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resource.qrc">
    <normaloff>:/img/icon.ico</normaloff>:/img/icon.ico</iconset>
  </property>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="gamemode">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Game Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="chessbd">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Chessboard Size</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gamemodeCB">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>55</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>PVE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PVP</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="chessbdCB">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>55</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>9×9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11×11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13×13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>28×28</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="IPtitle">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Remote IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="PORTtitle">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Remote Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IPinput">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PORTinput">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="serverStatus">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Server: Paused</string>
   </property>
  </widget>
  <widget class="QLabel" name="clientStatus">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Not Connected</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restartBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Restart Server</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="reconnectBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Reconnect</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="switchBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Offline Mode</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="usrnameInput">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="IDtitle">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
